# Cornell_bioinfor_course_projects
This repository stores major course projects I tool in Cornell relating Bioinformatics/Computational biology. These course are: 
1) BTRY 6830 - Quantitative Genomics and Genetics. Course description: Description
A rigorous treatment of analysis techniques used to understand complex genetic systems. This course covers both the fundamentals and advances in statistical methodology used to analyze disease and agriculturally relevant and evolutionarily important phenotypes. Topics include mapping quantitative trait loci (QTLs), application of microarray and related genomic data to gene mapping, and evolutionary quantitative genetics. Analysis techniques include association mapping, interval mapping, and analysis of pedigrees for both single and multiple QTL models. Application of classical inference and Bayesian analysis approaches is covered and there is an emphasis on computational methods.
2) BTRY 6840 - Computational Genetics and Genomics. Course description: Computational methods for analyzing genetic and genomic data. Topics include sequence alignment, hidden Markov Models for discovering sequence features, motif finding using Gibbs sampling, phylogenetic tree reconstruction, inferring haplotypes, and local and global ancestry inference. Prior knowledge of biology is not necessary to complete this course. Prerequisites: BTRY 3010 and CS 2110 or equivalents.
3) PLBIO 6000Concepts and Techniques in Computational Biology. Course description: This course is geared towards graduate students and advanced biology undergraduates seeking a better understanding of computational biology. Lectures will be a combination of presentations, paper discussions and hands-on sessions. Labs and paper discussions will have a significant component of plant science, but students from non-plant fields are also encouraged to register. Students will learn to work in a Unix environment, code using Python/R, and deploy tools for genome assembly, RNA-seq data analysis, local and global sequence alignment, protein domain searching using Hidden Markov Models, phylogenetic reconstruction, metabolomic analysis, and machine learning. Lectures will cover the algorithmic concepts underlying popular tools. The students will also learn practical aspects of implementing these tools in their own research using facilities available at Cornell.
# **Cornell Bioinformatics Course Projects**

This repository stores major course projects I took at Cornell related to Bioinformatics/Computational Biology. These courses are:

---

### **1. BTRY 6830 - Quantitative Genomics and Genetics**  
**Course description:**  
A rigorous treatment of analysis techniques used to understand complex genetic systems. This course covers both the fundamentals and advances in statistical methodology used to analyze disease and agriculturally relevant and evolutionarily important phenotypes.  
Topics include:  
- Mapping quantitative trait loci (QTLs)  
- Application of microarray and related genomic data to gene mapping  
- Evolutionary quantitative genetics  

**Analysis techniques include:**  
- Association mapping  
- Interval mapping  
- Analysis of pedigrees for both single and multiple QTL models  

The course also emphasizes computational methods and covers classical inference and Bayesian analysis approaches.

---

### **2. BTRY 6840 - Computational Genetics and Genomics**  
**Course description:**  
Computational methods for analyzing genetic and genomic data.  

**Topics include:**  
- Sequence alignment  
- Hidden Markov Models for discovering sequence features  
- Motif finding using Gibbs sampling  
- Phylogenetic tree reconstruction  
- Inferring haplotypes  
- Local and global ancestry inference  

**Prerequisites:**  
- BTRY 3010 and CS 2110 or their equivalents.  
(Note: Prior knowledge of biology is not necessary to complete this course.)

---

### **3. PLBIO 6000 - Concepts and Techniques in Computational Biology**  
**Course description:**  
This course is geared towards graduate students and advanced biology undergraduates seeking a better understanding of computational biology.  

**Course structure:**  
Lectures combine presentations, paper discussions, and hands-on sessions. Labs and discussions often focus on plant science, but students from non-plant fields are also encouraged to register.  

**Students will learn to:**  
- Work in a Unix environment  
- Code using Python/R  
- Deploy tools for genome assembly, RNA-seq data analysis, sequence alignment, protein domain searching (Hidden Markov Models), phylogenetic reconstruction, metabolomic analysis, and machine learning  

**Additional details:**  
Lectures cover the algorithmic concepts underlying popular tools. Students will also learn practical aspects of implementing these tools in their own research using Cornell facilities.
